newIDE/app/src/AssetStore/AssetsList.js

```diff
@@ -202,9 +201,8 @@ const PageBreakNavigation = ({
           }}
           disabled={pageBreakIndex <= 0}
         />
-        <RaisedButton
+        <FlatButton
```

Comment:
that button was too visible in the shop, where it's only to navigate pages. I think it's better this way

Reply:
Great